models/model1_part1.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # removes HTML tags and removes punctuation from the text
</pre><p>Move this comment inside the function.</p><p> </p><p>Use block quote <code>"""</code> to enclose comments </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>